docs/specs/files/41-ingressable-wiki-file.md

```diff
@@ -0,0 +1,71 @@
+---
+id: ingressable-wiki-file
+sidebar_label: Ingressable Wiki File
+date: 20240119
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl'
+import APITable from '@site/src/components/APITable';
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Ingressable Wiki File
```

Comment:
I suggest to extend the specifications of an ingressable wiki file to not only include the required contents (such as "# Title" or so), but also include optional syntax that makes things much handier. For example, I can see optional syntax for declaring, e.g., [formphrases](https://tno-terminology-design.github.io/tev2-specifications/docs/tev2-glossary#form-phrase-for-a-term). 

I can also think that in the ToIP CTWG there could be sessions with people that would actually like to generate glossaries, where they discuss what it is they actually need (of which I could think 'formphrases' are one).